npm/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@unsplash/intlc",
+  "version": "1.0.0-beta",
```

Comment:
Should this not match the version of intlc itself?

e.g. binary 0.2.0, npm 0.2.0-1 (what's idiomatic in npm for this? the `-n` suffix is what's used for Arch/AUR packages)

Reply:
That's what I thought first but then I figured having separate versions might give us more flexibility because let's say we have to fix a bug in the npm package, we could just bump that version without having to bump `intlc` itself?

Reply:
That's what the `-n` suffix is for. You start at `-1` and increment whenever you need to update the package independent of what you're packaging from upstream. When you update the upstream package you reset the counter.

Example timeline of versions:

- binary 0.2.0
- npm 0.2.0-1
- npm 0.2.0-2
- binary 0.2.1
- npm 0.2.1-1
- etc